params/JoCookSpecies.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 0.24
+    
+# e.g. "anas_krystallinus"    
+species.name <- JoCookSpecies
```

Comment:
Hey Jo, this need to be a character string. Can you correct and re-commit and push the file.